public/components/layout.js

```diff
@@ -44,6 +46,26 @@ class GeoHeader extends HTMLElement {
 
 customElements.define("geo-header",GeoHeader)
 
+class GeoNav extends HTMLElement {
+    #navTmpl = `
+    <link rel="stylesheet" href="https://unpkg.com/chota@latest">
+    <link rel="stylesheet" href="stylesheets/style.css">
+
+    <nav class="nav-center" style="justify-content: center; position: relative;"}>
+        <a style="margin: 0 10px;" class="active" href="index.html"> Home </a>
```

Comment:
`Home` suffices for finding links into navPlace.html and annotate.html.  Is there a reason you didn't include those pages in the nav?